lib/dialogs/create-issue.js

```diff
@@ -0,0 +1,34 @@
+module.exports = (message, channel, team) => {
+  const linkToMessage = `https://${team.domain}.slack.com/archives/${
```

Comment:
would be awesome if we could re-use existing logic for this by extracting the common code into a module